app/src/main/java/pl/futurecollars/invoicing/services/FileService.java

```diff
@@ -38,4 +54,12 @@ public void overwriteTheFile(final String line, StandardOpenOption openOption) {
             throw new FileSystemException("File system processing error.");
         }
     }
+
+    public void clear() {
+        try {
+            Files.write(invoicesDbPath, "".getBytes());
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Radek adviced to to avoid e.printStackTrace, mayby throw exception or log some message with log.error.

Reply:
Great catch, I missed that

Reply:
Yes, that's right. Corrected.